virttest/qemu_vm.py

```diff
@@ -1011,6 +1011,8 @@ def add_memorys(devices, params):
                 name = output[0]
                 backend_options = {}
                 backend_options["size_mem"] = "%sM" % params["mem"]
+                if params.get("backend_mem"):
```

Comment:
For having a consistent design with other machine-level mem relevant params (like `vm_mem_share`) I'd suggest using the name `vm_mem_backend` for the param instead.

Reply:
"backend_mem" is the key in using. if i use an new key does it mean all relevant cases need to be changed?

Reply:
> "backend_mem" is the key in using. if i use an new key does it mean all relevant cases need to be changed?

Actually, no. From my PoV, It just means we are introducing a new param to the **machine-level mem device** for specifying its backend type. Other mem devices would still use the `backend_mem` param.

Reply:
Agree,updated